scripts/mirror-images.sh

```diff
@@ -119,46 +104,40 @@ function dir_to_registry() {
         local -r img_pull_file=$1
         local -r local_dir=$2
         local -r dest_registry=$3
-        local -r image_tag=$4
-        local -r image_cnt=$(cut -d' ' -f1 <<< "$5")
-        local -r src_manifest=$(cut -d' ' -f2 <<< "$5")
+        local -r src_manifest=$4
 
         # Remove the manifest.json file name
         local src_img
         src_img=$(dirname "${src_manifest}")
         # Add the target registry prefix and remove SHA
-        local dst_img
-        dst_img="${dest_registry}/${src_img}"
-        dst_img=$(echo "${dst_img}" | awk -F'@' '{print $1}')
+        local -r dst_img="${dest_registry}/${src_img}"
+        local -r dst_img_no_tag="${dest_registry}/$(echo "${src_img}" | awk -F'@|:' '{print $1}')"
```

Comment:
Can it be done using bash string manipulation, like:
```
result=${input_string%%[@:]*}
```

Reply:
Done